### ISSUE TITLE
Can't confirm an email address using mail client's embedded IE browser

### ISSUE THREAD

**Issue by mamacdon** [mozilla/persona#2566]
My e-mail client is a corporate deployment of Lotus Notes 8.5, which uses (on Windows platforms) an embedded Internet Explorer 8 as its built-in web browser. When I try to confirm a Persona email address from Lotus Notes, the embedded browser fails to load the confirmation page and my email address is not confirmed.

Steps are:
1. Add a new email address to my Persona profile.
2. In Lotus Notes, open the confirmation email.
3. Click the "Click to confirm this email address" link.
4. I'm taken to a page and see a script error:

```
An error has occurred in the script on this page.
Line: 8
Char: 14823
Error: 'JSON' is undefined
Code: 0
URL: https://static.login.persona.org/v/53e7265f01/production/en_US/browserid.js
```

This probably happens because the embedded browser is running in compatibility mode. Setting an 'X-UA-Compatible' <meta> tag in the confirmation page should fix it.

**Comment by jrgm** [comment]
Thanks for the bug report. A couple of questions:
1. When you say "Add a new email address.." ... Did you do this in some other browser than the Lotus Notes embedded one? 
2. I downloaded Note 8.5.3 on Windows/XP and the embedded IE says it is "Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1; Trident/4.0; .NET CLR 1.1.4322; .NET CLR 2.0.50727; .NET CLR 3.0.04506.30; .NET CLR 3.0.04506.648; .NET CLR 3.0.4506.2152; .NET CLR 3.5.30729; .NET4.0C; .NET4.0E)". Are you sure that your embedded IE is IE8?

That's IE7, and we do not support that (except to nicely inform the user of that fact). The verify_email_address and reset_password pages though are not very nice in IE7.

**Comment by callahad** [comment]
We should totally be using the `<meta http-equiv="X-UA-Compatible" content="IE=Edge"/>` meta tag everywhere.

**Comment by mamacdon** [comment]
1. Yes -- I used Google Chrome to add the email address to my Persona account. Lotus Notes only came into play when I checked my email, and tried to click the confirmation link. (The problematic part of the above steps is from Step 2 onward.)
2. No, I was wrong to assume the embedded browser was IE8. My user-agent string is:

```
Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.1; WOW64; Trident/5.0; SLCC2; .NET CLR 2.0.50727; .NET CLR 3.5.30729; .NET CLR 3.0.30729; Media Center PC 6.0; .NET4.0C)
```

So If my understanding of this ( http://msdn.microsoft.com/en-us/library/ms537503%28v=vs.85%29.aspx ) is correct, my embedded browser is IE9 running in IE7 compatibility mode (as shown by the Trident/5.0 token). And yours is IE8 running in IE7 compatibility mode.

So in both of our cases, we actually are running supported browsers, but Persona's web pages have to follow callahad's suggestion to kick the embedded browser out of compatibility mode so we get JSON support.
